middleware/getPhotoOfTheDay.js

```diff
@@ -0,0 +1,24 @@
+const fetch = require("node-fetch");
+const API_KEY = require("../apiKey");
+const BASE_URL = "https://api.nasa.gov/planetary/apod";
+
+const checkStatus = (response) => {
+  if (response.ok) {
+    return response;
+  } else {
+    throw new Error("error");
+  }
+};
+
+const getPhotoOfTheDay = async (request, response, next) => {
+  const url = `${BASE_URL}?api_key=${API_KEY}`;
+  try {
+    const apodResponse = await fetch(url).then(checkStatus);
+    const apodJson = await apodResponse.json();
```

Comment:
I'd be tempted to put together these two lines into something like `await fetch(url).then(checkStatus).json()` but this is a very minor thing that has a lot of personal preference attached.

Reply:
Ah I did consider this, although I slightly lean more towards having these both on the two lines as individual constants as I think this makes the readability a bit more clearer.